src/app/components/PropertyMap.tsx

```diff
@@ -328,17 +328,41 @@ const PropertyMap: FC<PropertyMapProps> = ({
             closeOnClick={false}
             onClose={() => setPopupInfo(null)}
           >
-            <div>
-              <p className="font-semibold text-md p-1">
-                {popupInfo.feature.address}
-              </p>
+            <div
+              style={{
+                display: 'flex',
+                justifyContent: 'space-between',
+                flexWrap: 'nowrap',
+              }}
+            >
+              <div>
+                <p className="font-semibold text-md p-1">
+                  {popupInfo.feature.address}
+                </p>
+              </div>
+              <button onClick={() => setPopupInfo(null)}>
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  fill="none"
+                  viewBox="0 0 24 24"
+                  strokeWidth={1.5}
+                  stroke="currentColor"
+                  className="w-4 h-4"
+                >
+                  <path
+                    strokeLinecap="round"
+                    strokeLinejoin="round"
+                    d="M6 18 18 6M6 6l12 12"
+                  />
+                </svg>
```

Comment:
any reason you didn't use a Phosphor Icon as we used across the app here? if not tho, I'd prefer to take raw svg's and label them as a component so it's more easily parsable in the future (eg: adding `const XIcon = () => <svg>...` and then using `<XIcon />` here)

Reply:
No reason in particular. I wasn't sure of the global library we were using and `Favicon` is always good boilerplate. I'm using an icon from `Phosphur` now.